pkg/sfu/forwarder.go

```diff
@@ -1330,33 +1295,30 @@ func (f *Forwarder) Pause(brs Bitrates) VideoAllocation {
 
 	switch {
 	case f.muted:
-		alloc.state = VideoAllocationStateMuted
+		alloc.pauseReason = VideoPauseReasonMuted
 
 	case f.pubMuted:
-		alloc.state = VideoAllocationStatePubMuted
+		alloc.pauseReason = VideoPauseReasonPubMuted
 
-	case len(f.availableLayers) == 0:
-		// feed is dry
-		alloc.state = VideoAllocationStateFeedDry
+	case f.getOptimalBandwidthNeeded(brs, f.maxLayers) == 0:
+		// some exempt layer may be available, but technically the feed is dry
+		alloc.pauseReason = VideoPauseReasonFeedDry
 
 	default:
-		// feed bitrate is not yet calculated or pausing due to lack of bandwidth
-		alloc.state = VideoAllocationStateDeficient
+		// pausing due to lack of bandwidth
+		alloc.isDeficient = true
+		alloc.pauseReason = VideoPauseReasonBandwidth
 	}
 
 	f.clearParkedLayers()
 	return f.updateAllocation(alloc, "pause")
 }
 
 func (f *Forwarder) updateAllocation(alloc VideoAllocation, reason string) VideoAllocation {
-	if !f.targetLayers.IsValid() && alloc.targetLayers.IsValid() {
-		alloc.change = VideoStreamingChangeResuming
-	} else if f.targetLayers.IsValid() && !alloc.targetLayers.IsValid() {
-		alloc.change = VideoStreamingChangePausing
-	}
-
-	if alloc.state != f.lastAllocation.state || alloc.targetLayers != f.lastAllocation.targetLayers {
-		f.logger.Debugw(fmt.Sprintf("stream allocation: %s", reason), "allocation", alloc)
+	if alloc.isDeficient != f.lastAllocation.isDeficient ||
+		alloc.pauseReason != f.lastAllocation.pauseReason ||
+		alloc.targetLayers != f.lastAllocation.targetLayers {
+		f.logger.Infow(fmt.Sprintf("stream allocation: %s", reason), "allocation", alloc)
```

Comment:
switching this back to `Infow` as we have some reports of stream not starting. So, want to check those for a bit.